module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -544,6 +559,20 @@ protected function isHoldable($locationName)
         return !in_array($locationName, $excludeLocs);
     }
 
+    /**
+     * Check whether an item is holdable based on any current loan
+     *
+     * @param \stdClass $currentLoan The current loan
+     *
+     * @return bool
+     */
+    protected function isHoldableByCurrentLoan(\stdClass $currentLoan)
+    {
+        $currentLoanPatronGroup = $currentLoan->patronGroupAtCheckout->name;
```

Comment:
Is there any possibility this might be undefined? Should we add a fallback like `?? ''` to be on the safe side, or do we _want_ to cause an error if something is missing?

Reply:
Yes, but I see I broke the functionality of this PR in the latest merge from dev.  Will get that fixed before addressing this one.

Reply:
Ok fixed this -- yes a null check is a good idea just in case, as unlikely as it seems.